packages/core/src/tools/timeUtils.ts

```diff
@@ -0,0 +1,44 @@
+import { isNumber, round } from './utils'
+
+export type Duration = number & { d: 'Duration in ms' }
+export type ServerDuration = number & { s: 'Duration in ns' }
```

Comment:
Can we make this more explicit? it's not obvious that `ServerDuration` means a duration in ns. What about `NsDuration`? maybe we can also name the other `MsDuration`.

Reply:
> it's not obvious that ServerDuration means a duration in ns.

IMO it is more interesting to be explicit about when we should use this type rather than its content.

IMO, since the js APIs return ms, prefixing with `ms` or something like `internal` would add some unnecessary noise to the Duration type.

Reply:
Ok, makes sense.